cmd/state/generate/seeder.go

```diff
@@ -0,0 +1,120 @@
+package generate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	lg "github.com/anacrolix/log"
+	"github.com/anacrolix/torrent"
+	"github.com/anacrolix/torrent/bencode"
+	"github.com/anacrolix/torrent/metainfo"
+	"github.com/ledgerwatch/turbo-geth/common"
+	"github.com/ledgerwatch/turbo-geth/log"
+	trnt "github.com/ledgerwatch/turbo-geth/turbo/torrent"
+	"os"
+	"os/signal"
+	"time"
+)
+
+func Seed(pathes []string) error {
```

Comment:
What is the difference between this and `SeedSnapshot` in `remote_seeder` ?

Reply:
SeedSnapshot is for start seeding from scratch. remote_seeder can be used to download snapshots and continue seeding.